proto/addressmetadata.proto

```diff
@@ -1,39 +1,42 @@
 syntax = "proto3";
 package models;
 
+// Basic key/value used to store header data.
 message Header {
-    string Name = 1;
-    string Value = 2;
+    string name = 1;
+    string value = 2;
 }
 
-// Row is an indidual piece of structured data provided by wallet authors.
-// The headers help wallets determine how to deserialize the data for their purposes.
+// MetadataField is an indidual piece of structured data provided by wallet authors.
 message MetadataField {
-    repeated Header Headers = 1;
-
-    bytes Metadata = 2;
+    // The headers help wallets determine how to deserialize the data.
+    repeated Header headers = 1;
+    // Body of the metadata field.
+    bytes metadata = 2;
 }
 
-// Message is the user-specified data section of a AddressMetadata that is covered by the Signature provided by the user
+// Payload is the user-specified data section of a AddressMetadata that is covered by the users signature.
 message Payload {
     // Timestamp allows servers to determine which version of the data is the most recent.
-    int64 Timestamp = 1;
-    // User specified data.  Presumably some conventional data determined by wallet authors
-    repeated MetadataField Rows = 2;
+    int64 timestamp = 1;
+    // TTL tells us how long this entry should exist before being considered invalid.
+    int64 ttl = 2;
+    // User specified data.  Presumably some conventional data determined by wallet authors.
+    repeated MetadataField rows = 3;
 }
 
-// AddressMetadata is the basic unit of the keytp server.  It is used in both PUT and GET requests for various keyids
+// AddressMetadata is the basic unit of the keyserver.  It is used in both PUT and GET requests.
 message AddressMetadata  {
-    // Serialized version of the XPubKey.  The *hash* of this XPub should correspond to the `key` in the kv store
-    bytes PubKey = 1;
-    // Signature is the signature of the metadata by XPubKey
-    bytes Signature = 2;
-    // Signature type provided.  Default is Schnorr, but can be ecdsa
-    enum SignatureType {
-        Schnorr = 0;
+    // Serialized version of the XPubKey.  The *hash* of this XPub should correspond to the `key` in the kv store.
+    bytes pub_key = 1;
+    // Signature is the signature of the metadata by XPubKey.
+    bytes signature = 2;
+    // Signature scheme provided.  Default is Schnorr, but can be ecdsa.
+    enum SignatureScheme {
+        SCHNORR = 0;
```

Comment:
Why did you capitalize this?

Reply:
It seems a bit ugly to me too but
"Use CamelCase (with an initial capital) for enum type names and CAPITALS_WITH_UNDERSCORES for value names: "
from https://developers.google.com/protocol-buffers/docs/style